src/lib/expansion-panel/expansion-panel.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  Component,
+  Directive,
+  EventEmitter,
+  Input,
+  Output,
+  ViewEncapsulation,
+  AfterContentInit,
+  OnDestroy,
+  ContentChildren,
+  QueryList,
+  Optional,
+  forwardRef,
+} from '@angular/core';
+import {
+  trigger,
+  state,
+  style,
+  transition,
+  animate,
+} from '@angular/animations';
+import {Subscription} from 'rxjs/Subscription';
+
+/** Used to generate unique ID for each expansion panel. */
+let nextId = 0;
+/** Time and timing curve for expansion panel animations. */
+export const EXPANSION_PANEL_ANIMATION_TIMING = '225ms cubic-bezier(0.4,0.0,0.2,1)';
+
+@Directive({
+  selector: 'md-expansion-panel-set, mat-expansion-panel-set',
+  host: {
+    '[class.mat-expansion-panel-set]': 'true',
```

Comment:
You can just use a static class here, e.g.:
```ts
host: {'class': 'mat-expansion-panel-set'},
```
So long as it's not a `'[class]'` binding (which will clobber other classes)